www/include/Administration/performance/viewData.php

```diff
@@ -244,36 +248,38 @@
             $metric .= "(" . $metrics["unit_name"] . ")";
         }
     }
-    $index_data["metrics_name"] = $metric;
-    $index_data["service_description"] = "<a href='./main.php?p=50119&o=msvc&index_id=" . $index_data["id"] . "'>" .
-        $index_data["service_description"] . "</a>";
-
-    $index_data["storage_type"] = $storage_type[$index_data["storage_type"]];
-    $index_data["must_be_rebuild"] = $yesOrNo[$index_data["must_be_rebuild"]];
-    $index_data["to_delete"] = $yesOrNo[$index_data["to_delete"]];
-    $index_data["trashed"] = $yesOrNo[$index_data["trashed"]];
-    $index_data["hidden"] = $yesOrNo[$index_data["hidden"]];
-
-    if (isset($index_data["locked"])) {
-        $index_data["locked"] = $yesOrNo[$index_data["locked"]];
+    $indexData["metrics_name"] = $metric;
+    $indexData["service_description"] = "<a href='./main.php?p=50119&o=msvc&index_id=" . $indexData["id"] . "'>" .
+        $indexData["service_description"] . "</a>";
+
+    $indexData["storage_type"] = $storage_type[$indexData["storage_type"]];
+    $indexData["must_be_rebuild"] = $yesOrNo[$indexData["must_be_rebuild"]];
+    $indexData["to_delete"] = $yesOrNo[$indexData["to_delete"]];
+    $indexData["trashed"] = $yesOrNo[$indexData["trashed"]];
+    $indexData["hidden"] = $yesOrNo[$indexData["hidden"]];
+
+    if (isset($indexData["locked"])) {
+        $indexData["locked"] = $yesOrNo[$indexData["locked"]];
     } else {
-        $index_data["locked"] = $yesOrNo[0];
+        $indexData["locked"] = $yesOrNo[0];
     }
 
-    $index_data["class"] = $tab_class[$i % 2];
-    $data[$i] = $index_data;
+    $indexData["class"] = $tab_class[$i % 2];
+    $data[$i] = $indexData;
 }
 
-/* Get the list of running poller */
-$queryPollers = "SELECT instance_id, name FROM instances ORDER BY name";
-$instances = array();
-try {
-    $res = $pearDBO->query($queryPollers);
-} catch (\PDOException $e) {
-    while ($row = $res->fetchRow()) {
-        $instances[$row['instance_id']] = $row['name'];
-    }
-}
+//select2 Poller
+$poller = $searchP;
```

Comment:
https://github.com/centreon/centreon/pull/8832/files#diff-df1329e0e01c6ef2185ebe632f821f8aR123
$searchP could be empty in the request. Shouldn't you add a default value, like '' : https://github.com/centreon/centreon/pull/8832/files#diff-df1329e0e01c6ef2185ebe632f821f8aL404